benchmark/collective/bert/train.sh

```diff
@@ -22,6 +22,12 @@ if [[ $# -ge 1 ]]; then
   distributed_args="--cluster_node_ips ${cluster_node_ips} --node_ip ${node_ip}"
 fi
 
+export FLAGS_fuse_parameter_memory_size=64 #MB
```

Comment:
i found fuse memory size here, why not put it to distributed strategy?

Reply:
It's now controlled by flags instead of buildstrategy's option.